model/components/nagios.yml

```diff
@@ -0,0 +1,71 @@
+---
+components:
+
+# Nagios Core
+- id: nagios-core
+  name: Nagios Core
+  description: Nagios is an open source host, service and network monitoring program. It monitors specified hosts and services, alerting you to any developing issues, errors or improvements
+  link: https://www.nagios.org/projects/nagios-core/
+  github:
+    - NagiosEnterprises/nagioscore
+  twitter: nagiosinc
+  logo: nagios
+  license:
+    - GPLv2
+  categories:
+    - storage
+    - visualization
+    - alerting
+    - dashboarding
+  capabilities:
+    aspects:
+      - metrics [host]
+      - metrics [app]
+      - metrics [process]
+  connections:
+    - nagios-rdp
+    - nagios-cpa
+
+# Nagios Cross-Platform Agent
+- id: nagios-cpa
+  name: Nagios Cross-Platform Agent
+  description: The Nagios Cross-Platform Agent is a single monitoring agent that allows both active checks via check_ncpa.py and passive checks via NRDP. NCPA comes with a built-in web GUI, documentation, websocket graphing, and is secured with SSL by default.
+  link: https://www.nagios.org/ncpa/
+  github:
+    - NagiosEnterprises/ncpa
+  twitter: nagiosinc
+  logo: nagios
+  license:
+    - Nagios Open Software License Version 1.3
+  categories:
+    - agent
+    - dashboarding
```

Comment:
Maybe remove dashboarding here, since it's not a full fletched dashboard that the agent provides, or?